utils/participantDataCleanup.js

```diff
@@ -0,0 +1,24 @@
+const { getResponseJSON } = require('./shared');
+
+const participantDataCleanupAPIs = async (req, res) => {
```

Comment:
```suggestion
const participantDataCleanup = async (req, res) => {
```

Reply:
For event-triggered functions, there'll be no http requests passed on.

Reply:
Solved